### ISSUE TITLE
Django-guardian incompatibility

### ISSUE THREAD

**Issue by larwaa** [rubberdok/indok-web#126]
**TL;DR:** Django-guardian is more incompatible than anticipated and is likely not a good solution. Need to figure out how to solve per object permissions 🙃

#### Description
Django-guardian's custom implementation of `AnonymousUser` likely causes more incompatibility issues than previously anticipated. Graphql-JWT uses the standard Django `AnonymousUser` for its authentication needs, and will therefore cause compatibility issues with certain functions and operations of `django-guardian`.

#### Example
```python
from guardian.shortcuts import assign_perm
assign_perm("permission", info.context.user, obj)
```

Where `info.context.user` is the standard Django anonymous user if the user making the request is not authenticated, and will cause an unhandled exception as `django-guardian` assumes the anonymous user to be an actual instance of type User.

#### Going forward
While I am yet to determine the extent of the incompatibility, I believe the decision to implement `django-guardian` unfortunately was based on an incomplete assessment of the compatibility of the module, and do not believe it to be a viable solution for row level permissions. As such, I will be making a PR to revert #75 (edit: see #127).

While it is possible to solve this issue by manually checking if certain requirements are met before returning restricted data, I believe it to be a poor long term solution which will cause maintainability issues and increase the risks of broken access control. Therefore, I would gladly accept any suggestions on how to solve per object permissions.
